.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.5.4
+current_version = 2.55
```

Comment:
I think you missed the dot between the 5s.

Reply:
fixed